content/docs/200-inputs/600-multi-select.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Multi-select
+excerpt: Select from a large list of multi-select options.
+---
+
+Add a _multi-select_ input to your app to allow users select from a large list of multi-select options.
```

Comment:
```suggestion
Add a _multi-select_ input to your app to allow users select multiple values from a list of options.
```

Just a suggestion to remove the "multi-select" repetition. Also maybe drop the word "large" here. I used "large" in the Select description to differentiate it from Button Group but in this case there's only the Multi-select input when you want to select multiple values.